client/src/app/site/projector/services/current-list-of-of-speakers-slide.service.ts

```diff
@@ -19,13 +43,64 @@ export class CurrentListOfSpeakersSlideService {
         };
     }
 
+    /**
+     * Returns an observable for the agenda item id of the currently projected element on the
+     * given projector.
+     *
+     * @param projector The projector to observe.
+     * @returns An observalbe for the agenda item id. Null, if no element with an agenda item is shown.
+     */
+    public getAgendaItemIdObservable(projector: ViewProjector): Observable<number | null> {
```

Comment:
@emanuelschuetze @FinnStutzenstein 
I fear it might become too confusing.
From the point of view of somebody who only wants to manage the current speaker(s), getting the CLOS should be straight forward without the need of selecting a projector.

Reply:
@emanuelschuetze ping

Reply:
We'll test and improve it in a next PR...